style.css

```diff
@@ -0,0 +1,107 @@
+canvas{
+    image-rendering: pixelated;
+    margin-left: 20px;
+}
+
+header{
+    border-bottom:1px black solid;
+}
+div{
+    display: flex;
+    margin-top: 40px;
+}
```

Comment:
too dangerous
all `div` on the page will have these props, no matter where they are